packages/client/lib/rpc/util/CLConnectionManager.ts

```diff
@@ -113,44 +126,54 @@ export class CLConnectionManager {
   }
 
   private _getPayloadLogMsg(payload: NewPayload) {
-    const msg = `number=${Number(
-      payload.payload.blockNumber
-    )} hash=${payload.payload.blockHash.substring(0, 7)}... parentHash=${
-      payload.payload.parentHash
-    } status=${payload.response ? payload.response.status : '-'}`
+    const msg = `number=${Number(payload.payload.blockNumber)} hash=${short(
+      payload.payload.blockHash
+    )} parentHash=${short(payload.payload.parentHash)}  status=${
+      payload.response ? payload.response.status : '-'
+    } gasUsed=${this.compactNum(Number(payload.payload.gasUsed))} baseFee=${Number(
+      payload.payload.baseFeePerGas
+    )} txs=${payload.payload.transactions.length}`
     return msg
   }
 
   private _getForkchoiceUpdateLogMsg(update: ForkchoiceUpdate) {
-    let msg = `headBlockHash=${update.state.headBlockHash.substring(0, 7)}...`
+    let msg = ''
+    if (update.headBlock) {
+      msg += `number=${Number(update.headBlock.header.number)} `
+    }
+    msg += `head=${short(update.state.headBlockHash)} finalized=${short(
+      update.state.finalizedBlockHash
+    )} response=${update.response ? update.response.payloadStatus.status : '-'}`
     if (update.headBlock) {
-      msg += ` number=${update.headBlock.header.number} timestampDiff=${this.timeDiffStr(
-        update.headBlock
-      )}`
+      msg += ` timestampDiff=${this.timeDiffStr(update.headBlock)}`
     }
-    msg += ` finalizedBlockHash=${update.state.finalizedBlockHash} response=${
-      update.response ? update.response.payloadStatus.status : '-'
-    }`
     if (update.error) {
       msg += ` error=${update.error}`
     }
     return msg
   }
 
+  private compactNum(num: number) {
+    return this.numberFormatter.format(num).toLowerCase()
+  }
+
   private timeDiffStr(block: Block) {
     const timeDiff = new Date().getTime() / 1000 - block.header.timestamp.toNumber()
     const min = 60
     const hour = min * 60
     const day = hour * 24
     let timeDiffStr = ''
     if (timeDiff > day) {
-      timeDiffStr = `${Math.floor(timeDiff / day)} days`
+      timeDiffStr = `${Math.floor(timeDiff / day)} day`
```

Comment:
Who needs `moment` when we can do it ourselves?

Reply:
I hated this but I didn't want to add an extra dependency. 🙂 (or should we do in this case? This *would* likely be somewhat handy for some several use cases.)

Reply:
I kind of like it and its simplicity! if we want to use it elsewhere, we can move it to a util file. much better than a whole new dependency for these ~15 lines :)